sycl/source/detail/builtins_helper.hpp

```diff
@@ -153,44 +161,51 @@
   }
 
 #define MAKE_1V_2S(Fun, Ret, Arg1, Arg2)                                       \
+  __MAKE_1V_2S(Fun, 1, Ret, Arg1, Arg2)                                        \
   __MAKE_1V_2S(Fun, 2, Ret, Arg1, Arg2)                                        \
   __MAKE_1V_2S(Fun, 3, Ret, Arg1, Arg2)                                        \
   __MAKE_1V_2S(Fun, 4, Ret, Arg1, Arg2)                                        \
   __MAKE_1V_2S(Fun, 8, Ret, Arg1, Arg2) __MAKE_1V_2S(Fun, 16, Ret, Arg1, Arg2)
 
 #define MAKE_1V_2S_3S(Fun, Ret, Arg1, Arg2, Arg3)                              \
+  __MAKE_1V_2S_3S(Fun, 1, Ret, Arg1, Arg2, Arg3)                               \
   __MAKE_1V_2S_3S(Fun, 2, Ret, Arg1, Arg2, Arg3)                               \
   __MAKE_1V_2S_3S(Fun, 3, Ret, Arg1, Arg2, Arg3)                               \
   __MAKE_1V_2S_3S(Fun, 4, Ret, Arg1, Arg2, Arg3)                               \
   __MAKE_1V_2S_3S(Fun, 8, Ret, Arg1, Arg2, Arg3)                               \
   __MAKE_1V_2S_3S(Fun, 16, Ret, Arg1, Arg2, Arg3)
 
 #define MAKE_SR_1V_AND(Fun, Call, Ret, Arg1)                                   \
+  __MAKE_SR_1V_AND(Fun, Call, 1, Ret, Arg1)                                    \
   __MAKE_SR_1V_AND(Fun, Call, 2, Ret, Arg1)                                    \
   __MAKE_SR_1V_AND(Fun, Call, 3, Ret, Arg1)                                    \
   __MAKE_SR_1V_AND(Fun, Call, 4, Ret, Arg1)                                    \
   __MAKE_SR_1V_AND(Fun, Call, 8, Ret, Arg1)                                    \
   __MAKE_SR_1V_AND(Fun, Call, 16, Ret, Arg1)
 
 #define MAKE_SR_1V_OR(Fun, Call, Ret, Arg1)                                    \
+  __MAKE_SR_1V_OR(Fun, Call, 1, Ret, Arg1)                                     \
   __MAKE_SR_1V_OR(Fun, Call, 2, Ret, Arg1)                                     \
   __MAKE_SR_1V_OR(Fun, Call, 3, Ret, Arg1)                                     \
   __MAKE_SR_1V_OR(Fun, Call, 4, Ret, Arg1)                                     \
   __MAKE_SR_1V_OR(Fun, Call, 8, Ret, Arg1)                                     \
   __MAKE_SR_1V_OR(Fun, Call, 16, Ret, Arg1)
 
 #define MAKE_1V_2P(Fun, Ret, Arg1, Arg2)                                       \
+  __MAKE_1V_2P(Fun, 1, Ret, Arg1, Arg2)                                        \
   __MAKE_1V_2P(Fun, 2, Ret, Arg1, Arg2)                                        \
   __MAKE_1V_2P(Fun, 3, Ret, Arg1, Arg2)                                        \
   __MAKE_1V_2P(Fun, 4, Ret, Arg1, Arg2)                                        \
   __MAKE_1V_2P(Fun, 8, Ret, Arg1, Arg2) __MAKE_1V_2P(Fun, 16, Ret, Arg1, Arg2)
```

Comment:
Minor comment: it is not about your change, but shouldn't this line be split into 2 lines. Same for Line 168.

Reply:
I'd prefer to address this separately.